src/pages/costModels/costModel/updateDistributionDialog.tsx

```diff
@@ -1,11 +1,24 @@
-import { Alert, Button, Form, FormGroup, Modal, Radio, Stack, StackItem, Title } from '@patternfly/react-core';
+import {
+  Alert,
+  Button,
+  Form,
+  FormGroup,
+  Modal,
+  Radio,
```

Comment:
Missed the radio buttons in this dialog, but they should also be mutually exclusive (i.e., use the same name)

Reply:
Done